src/Swan.Lite/Extensions.Enumerable.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+
+namespace Swan
+{
+    /// <summary>
+    /// This class contains extensions methods for types implementing IEnumerable<T>
```

Comment:
"Extension" is not a name here, therefore it is plural-invariant: "extension methods".

Also, use `&lt;` instead of `<` and `&gt;` instead of `>`. Documentation comments are XML.

The type parameter is better called `TSource` instead of just `T`, to harmonize with `System.Linq.Enumerable` methods.